static/index.js

```diff
@@ -20,9 +25,9 @@ document.querySelectorAll('.scroll_indicator').forEach(scrollIndicator => {
     function scroll() {
         var top = window.pageYOffset || document.documentElement.scrollTop;
         if (top > 10) {
-            scrollIndicator.classList.add("hidden");
+            scrollIndicator.classList.add("scrolled");
```

Comment:
Is this change intentional? This got changed recently

Reply:
Oh, I will fix it